packages/react/src/docs/customization.stories.mdx

```diff
@@ -0,0 +1,108 @@
+import { Canvas, Meta, Story, Source, ArgsTable } from '@storybook/addon-docs';
```

Comment:
According to the PR (https://github.com/openfun/cunningham/pull/69), should we remove `stories` from the name of this file ?

Reply:
Indeed! This commit was created before, that's why :)